README.md

```diff
@@ -472,6 +472,48 @@ If an attenuated resource or capability is desired, it MUST be explicitly listed
 ]
 ```
 
+## 5.2 Top
+
+The "top" (or "super user") ability MUST be denoted `*/*`. The top ability grants access to all other capabilities for the specified resource, across all possible namespaces. Top corresponds to an "all" matcher, whereas [delegation](#51-ucan-delegation) corresponds to "any" in the UCAN chain. The top ability is useful when "linking" agents by delegating all access to resource(s). This is the most powerful ability, and as such it SHOULD be handled with care.
+
+```
+                                           ┌───────┐
+                                           │       │
+                                           │  */*  │
+                                           │       │
+                                           └▲──▲──▲┘
+                                            │  │  │
+               ┌────────────────────────────┘  │  └────────────────────┐
+               │                               │                       │
+          ┌────┴────┐                     ┌────┴─────┐             ┌───┴───┐
+          │         │                     │          │             │       │
+          │  msg/*  │                     │  crud/*  │             │  ...  │
+          │         │                     │          │             │       │
+          └─▲────▲──┘                     └─▲──────▲─┘             └───────┘
+            │    │                          │      │
+            │    │                          │      │
+            │    │                          │      │
+            │    │                          │      │
+┌───────────┴┐ ┌─┴─────────────┐ ┌──────────┴──┐ ┌─┴─────────────┐
+│            │ │               │ │             │ │               │
+│  msg/send  │ │  msg/receive  │ │  crud/read  │ │  crud/mutate  │
+│            │ │               │ │             │ │               │
+└────────────┘ └───────────────┘ └─────────────┘ └─▲─────▲─────▲─┘
+                                                   │     │     │
+                                            ┌──────┘     │     └──────┐
+                                            │            │            │
+                                            │            │            │
+                               ┌────────────┴──┐ ┌───────┴───────┐ ┌──┴─────────────┐
+                               │               │ │               │ │                │
+                               │  crud/create  │ │  crud/update  │ │  crud/destroy  │
+                               │               │ │               │ │                │
+                               └───────────────┘ └───────────────┘ └────────────────┘
+```
+
+### 5.2.1 Bottom
+
+In concept there is a "bottom" ability ("none" or "void"), but it is not possible to represent in an ability. As it is merely the absecnce of any ability, it is not possible to construct a capability with a bottom ability.
+
```

Comment:
I realized that we actually do need a top ability for a bunch of linking use cases at Fission (and I believe Bluesky), and that it likely does need to live in the core spec. I had removed it a few days ago after seeing how DAG House is approaching their modelling. This does not alter how they need to think about their abilities at all, and they do not need to adopt `*/*`.

I expanded on the explanation & diagram after explaining the feature to Irakli a few days ago. The syntax is also changed to be consistent with the `*/*` pattern that we see showing up in the wild.